joycond-cemuhook.py

```diff
@@ -133,9 +133,15 @@ def __init__(self, server, index, device, motion_device, motion_only=False):
         self.player_id = get_player_id(self.led_status)
 
         with open(os.path.join('profiles', self.name + '.json')) as profile:
-            self.keymap = {evdev.ecodes.ecodes[ecode]:ps_key for ecode,ps_key in json.load(profile).items()}
+            original_keymap = json.load(profile).items()
```

Comment:
It would be better to change this line to `original_keymap = json.load(profile)`, since most iterations don't use both the key and value (for example, line 137 only uses `ecode`, so use `original_keymap.values()` there, and line 143 only uses `ps_key`, so use `original_keymap.keys()` there)